hrms/patches.txt

```diff
@@ -8,4 +8,5 @@ hrms.patches.v14_0.create_custom_field_for_appraisal_template
 hrms.patches.post_install.update_performance_module_changes #2023-04-17
 hrms.patches.v14_0.update_ess_user_access #2023-08-14
 execute:frappe.db.set_default("date_format", frappe.db.get_single_value("System Settings", "date_format"))
-hrms.patches.v14_0.create_vehicle_service_item
\ No newline at end of file
+hrms.patches.v14_0.create_vehicle_service_item
+hrms.patches.v14_0.change_appointment_letter_introduction_to_texteditor #28th August 2023
```

Comment:
You just need to change field type by editing the doctype directly. Patch isn't required here.

Reply:
Ok. This happens for standard fields though. 
![image](https://github.com/frappe/hrms/assets/55623011/08c49778-ddfd-4fb3-ab55-1775484ca94d)

Reply:
Ohh okay. Actually, both the field types create a longtext column in DB. So this transition should be allowed IG.
Description is a Text Editor field. Introduction is Long Text

<img width="721" alt="image" src="https://github.com/frappe/hrms/assets/24353136/e581340f-84df-48d0-b0cd-3b63c66b4631">

But the quill editor adds a div enclosing the content. Maybe that's why the transition is not allowed. 

<img width="531" alt="image" src="https://github.com/frappe/hrms/assets/24353136/6c536195-df50-4bfc-be74-9daf5a4eb208">

Will have to check the framework. Closing this for now. Adding a patch won't help.